src/Splitio/Domain/RedisConfigurationValidator.cs

```diff
@@ -15,7 +15,14 @@ public static void Validate(CacheAdapterConfigurationOptions config)
             {
                 throw new ArgumentException("Redis config should be set to build split client in Consumer mode.");
             }
-        
+            if (!string.IsNullOrEmpty(config.ConnectionString))
+            {
+                if (!string.IsNullOrEmpty(config.Host) || config.RedisClusterNodes != null)
+                {
+                    _log.Warn("Redis Connection String is set, will ignore all other properties.");
+                }
+                return;
+            }
```

Comment:
You should override `KeyHashTag` here
```suggestion
            if (!string.IsNullOrEmpty(config.RedisConnectionString))
            {
                if (!string.IsNullOrEmpty(config.Host) || config.RedisClusterNodes != null)
                {
                    _log.Warn("Redis Connection String is set, will ignore all other properties.");
                }
            
                config.RedisClusterNodes = new ClusterNodes();
            
                SetKeyHashTagDefault(config);
            
                return;
            }
```

and you need to create the SetKeyHashTagDefault(config) method and reuse it:
```
private static void SetKeyHashTagDefault(CacheAdapterConfigurationOptions config)
{
    config.RedisClusterNodes.KeyHashTag = "{SPLITIO}";
    _log.Warn("Redis Cluster Hashtag is not set, will set its value to: {SPLITIO}.");
}
```